app/cdap/components/Connections/Create/ConnectionConfiguration/ConnectionConfigForm.tsx

```diff
@@ -58,6 +60,33 @@ export function ConnectionConfigForm({
   const [name, setName] = React.useState(initName);
   const [description, setDescription] = React.useState(initDescription);
   const classes = useStyle();
+  const [configurationErrors, setConfigurationErrors] = React.useState(
+    testResults.configurationErrors
+  );
+
+  const onCreate = () => {
+    const fieldErrors = {};
+    for (const [fieldName, value] of Object.entries(connectorProperties)) {
+      if (
+        !!value.required &&
+        (!values.hasOwnProperty(fieldName) || values[fieldName] === '' || values[fieldName] == null)
```

Comment:
should be `values[fieldName] === null` ?

Reply:
intentionally covering both undefined and null